core/src/main/java/com/microsoft/applicationinsights/internal/util/Sanitizer.java

```diff
@@ -168,7 +187,57 @@ private static String MakeKeyNonEmpty(String key) {
         return key;
     }
 
+    @Obsolete
```

Comment:
Do we need to mark private methods as deprecated? Is it for better internal usage and API tracking?

Reply:
I have marked private methods in this class as obsolete because public methods of this class use those private methods, so kind of I cannot remove it but we are no longer using it. Makes sense?